doc/overview/architecture.md

```diff
@@ -105,7 +104,7 @@ run one DAOS *server* instance per node, which in turn starts one
 DAOS *Engine* process per physical socket. Membership of the DAOS
 servers is recorded into the system map, that assigns a unique integer
 *rank* to each *Engine* process. Two different DAOS systems comprise
-two disjoint sets of DAOS servers, and do not coordinate with each other.
+two disjointed sets of DAOS servers, and do not coordinate with each other.
```

Comment:
disjoint is the accurate term here with regard to the set theory related meaning